roxctl/generate/README.md

```diff
@@ -0,0 +1,115 @@
+# Static Network Policy Generator
+
+## Developer Preview Notice
+
+The static network policy generation feature is offered as a _developer preview_ feature.
+While we are open to receiving feedback about this feature, our technical support will be able to
+assist and answer questions about it.
+
+## About
+
+The static network policy generator a tool analyzes k8s manifests and generates a set of suggested network policies in form of yaml documents that may be directly applied to a k8s cluster. It is integrated with [NP-Guard](https://np-guard.github.io/) _analyze & synthesize_ module that discovers the network connections and generates the network policies. For more details, refer to the [NP-Guard webpage](https://np-guard.github.io/).
```

Comment:
```suggestion
The static network policy generator a tool analyzes k8s manifests and generates a set of suggested network policies in form of yaml documents that may be directly applied to a k8s cluster. It is integrated with NP-Guards _cluster-topology-analyzer_ module that discovers the network connections and generates the network policies. For more details, refer to the [NP-Guard webpage](https://np-guard.github.io/).
```
I think it is called the Cluster Topology Analyzer, but I might be wrong

Reply:
Also, I've removed one of the NP-Guard links to not have the same link within one sentence distance two times

Reply:
Let me then link `Cluster Topology Analyzer` to its Github repo and keep the second link as it is.

Reply:
See [edfb373](https://github.com/stackrox/stackrox/pull/2637/commits/edfb373fe06d368440609f02b30e24003870cf9a)